netbox/dcim/tables/racks.py

```diff
@@ -66,6 +66,10 @@ class RackTypeTable(NetBoxTable):
         template_code="{{ record.outer_width }} {{ record.outer_unit }}",
         verbose_name=_('Outer Width')
     )
+    outer_height = tables.TemplateColumn(
```

Comment:
All the outer dimension columns for Racks and RackTypes display "None None" if the dimension is not set. We should take this opportunity to fix that.